src/audits/performance/critical-request-chains.js

```diff
@@ -63,7 +63,7 @@ class CriticalRequestChains extends Audit {
    */
   static audit(artifacts) {
     let chainCount = 0;
-    function walk(node, depth, startTime) {
+    function walk(node, depth) {
```

Comment:
Yes, Closure fails without this fix. That's one of the points of this
change :)

On Fri., 27 May 2016, 18:53 Paul Lewis, <notifications@github.com> wrote:

> In src/audits/performance/critical-request-chains.js
> <https://github.com/GoogleChrome/lighthouse/pull/360#discussion_r64875642>
> :
>
> > @@ -63,7 +63,7 @@ class CriticalRequestChains extends Audit {
> >     */
> >    static audit(artifacts) {
> >      let chainCount = 0;
> > -    function walk(node, depth, startTime) {
> > +    function walk(node, depth) {
>
> Mind you, it's fine actually... the fix needs to exist. Go for it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleChrome/lighthouse/pull/360/files/3bf74374fb5c3c808e3e29f6713ee6153dce370f#r64875642>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHRkKZ1azw0_yLRageNFSfcdLDo4A9Bks5qFrDtgaJpZM4InRU2>
> .
>